zap-api-go/zap/interface.go

```diff
@@ -1,264 +1,246 @@
-// Zed Attack Proxy (ZAP) and its related class files.
```

Comment:
This should be kept.

Reply:
The `interface.go` file is now automatically generated using a bash script. I will update the template to include the previous comments.